### ISSUE TITLE
OpenRCT2 Flatpak build is marked as DEBUG

### ISSUE THREAD

**Issue by Calinou** [flathub/io.openrct2.OpenRCT2#73]
As seen in the bottom-left corner of the main menu:

![image](https://github.com/flathub/io.openrct2.OpenRCT2/assets/180032/df787a6c-10cd-4a7b-b53f-57c8a1575962)

I assume this means the build is less optimized or performs debug checks, potentially decreasing performance. There's probably a switch to compile a release build instead, but I haven't investigated this.

**Comment by janisozaur** [comment]
This is a simple check for presence of assert macros:

https://github.com/OpenRCT2/OpenRCT2/blob/68575ba165b3143457f164c6ef5dead03c9049e5/src/openrct2/Version.cpp#L47

https://en.cppreference.com/w/c/error/assert

I checked current version of https://github.com/flathub/io.openrct2.OpenRCT2/blob/master/io.openrct2.OpenRCT2.yaml and there is no CMake build type set. We (OpenRCT2) don't set it by default either and I don't know how to check build logs. Launchpad (which builds using our debian packaging instructions and creates packages for PPA) sets build type through its own means and has no build type set explicitly either.

TL;DR: this message does not mean the build is unoptimized, but it is worth checking what build type is used for flathub builds.

**Comment by janisozaur** [comment]
I located where the logs should be: https://buildbot.flathub.org/#/builders/25/builds/8262/steps/8/logs/stdio but it is empty.

**Comment by janisozaur** [comment]
@rmnvgr can you provide any insight why the logs are empty and what is the cmake build type used?

**Comment by rmnvgr** [comment]
I have set the build type to Release for the latest version in #83.
